shell/browser/net/system_network_context_manager.cc

```diff
@@ -154,6 +156,9 @@ SystemNetworkContextManager::CreateDefaultNetworkContextParams() {
   network::mojom::NetworkContextParamsPtr network_context_params =
       network::mojom::NetworkContextParams::New();
 
+  content::UpdateCorsExemptHeader(network_context_params.get());
+  variations::UpdateCorsExemptHeaderForVariations(network_context_params.get());
```

Comment:
`variations::UpdateCorsExemptHeaderForVariations` is not needed, as chrome finch experiment is not supported by Electron.

Also can you add a comment that

`This is required to avoid blocking X-Requested-With headers sent by PPAPI plugins`

Reply:
All done